docs/extend/modular-resources/examples/odrive.md

```diff
@@ -0,0 +1,281 @@
+
+[Prepare](#prepare-your-odrive) your ODrive and [download](#requirements) and [configure](#configuration) the module to add an `odrive-serial` or `odrive-canbus` [motor](/components/motor/) {{< glossary_tooltip term_id="resource" text="resource" >}} to your robot.
+
+{{% alert title="Note" color="note" %}}
+
+This module is only implemented for use with the [Viam Python SDK](https://python.viam.dev/).
+The methods other SDKs provide for the [motor API](/components/motor#API) will not work with this module.
+
+{{% /alert %}}
+
+### Prepare your ODrive
+
+1. Read through the [ODrive documentation](https://docs.odriverobotics.com/v/latest/getting-started.html) to wire, calibrate, and configure your ODrive natively.
+
+{{% alert title="Tip" color="tip" %}}
+
+This configuration remains on the same ODrive motor controller across reboots, and only changes when you go through the configuration of the ODrive again.
+
+To set the native configuration dynamically, find and copy the path to the motor's `config.json` file to provide in configuration as the attribute `"odrive_config_file"`.
+See the [Viam Github](https://github.com/viamrobotics/odrive/tree/main/sample-configs) for an example of this file.
+
+{{% /alert %}}
+
+2. Follow [this guide](https://docs.odriverobotics.com/v/latest/control.html#control-doc) to tune your ODrive motor.
+
+3. See the [ODrive CAN documentation](https://docs.odriverobotics.com/v/latest/can-guide.html) for detailed information on how to set up CAN on your ODrive.
+
+{{% alert title="Tip" color="tip" %}}
+
+If you are using a Raspberry Pi as your [board](/components/board/), you must run `sudo ip link set can0 up type can bitrate <baud_rate>` in your terminal to receive CAN messages.
+See [Troubleshooting](#can-link-issues) for more information.
+
+Additionally, make sure you have [enabled SPI communication on your Pi](/installation/prepare/rpi-setup/) to use several common CANHats.
+
+{{% /alert %}}
+
+4. Make sure your ODrive is connected to your [board](/components/board/) as follows, depending on your preferred connection method:
+
+{{< tabs name="Connect your ODrive">}}
+{{% tab name="odrive-serial" %}}
+
+Plug the [USB Isolator for Odrive](https://odriverobotics.com/shop/usb-c-to-usb-a-cable-and-usb-isolator) into a USB port on your board.
+Plug a USB-C to USB-A cable from the isolator to the ODrive.
+
+{{% /tab %}}
+{{% tab name="odrive-canbus" %}}
+
+Wire the CANH and CANL pins from your board to your ODrive.
+Refer to your board and the [ODrive's pinout](https://docs.odriverobotics.com/v/latest/pinout.html) diagrams for the location of these pins.
+
+You will have already made a serial connection to set up your ODrive.
+After setting up the ODrive, if you wish to use the `odrive-canbus` model, you can either leave the serial connection plugged in or remove it and leave only the CANH and CANL pins wired.
```

Comment:
or maybe here is the best place to note that if you want to unplug the serial connection and only use the CAN pins, you can't have an `"odrive_config_file"` in the viam config

Reply:
Done!